server/migrations/phase-allocation-migrations/1661284647061_add-phase-tables.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable no-restricted-syntax, no-await-in-loop */
-const { dbClient, schema } = require('../db');
+const { dbClient, schema } = require('../../db');
```

Comment:
This should be in the other PR but if these are merged together I won't complain